tests/unit/mixins/data-adapter-mixin-test.js

```diff
@@ -43,10 +43,11 @@ describe('DataAdapterMixin', () => {
       expect(hash).to.have.ownProperty('beforeSend');
     });
 
-    it('asserts the presence of authorizer', function() {
+    it('asserts `authorize` is overridden', function() {
       adapter.set('authorizer', null);
       expect(function() {
         adapter.ajaxOptions();
+        hash.beforeSend();
       }).to.throw(/Assertion Failed/);
     });
 
```

Comment:
I think we should have a test for `authorize` actually being called when a request is made.

Reply:
👍